ucm2/USB-Audio/Realtek-ALC1220-VB-Desktop-HiFi.conf

```diff
@@ -0,0 +1,60 @@
+SectionDevice."Speaker" {
+        Comment "Speakers"
+        Value {
+               PlaybackChannels "8"
+               PlaybackPCM "hw:ALC1220VBDT,0"
+               JackControl "Speaker Jack"
+               PlaybackMixerElem "Speaker"
+        }
+}
+
+SectionDevice."Headphone" {
+        Comment "Front Headphone"
+        Value {
+                PlaybackChannels "2"
+                PlaybackPCM "hw:ALC1220VBDT,1"
+                JackControl "Front Headphone Jack"
+                PlaybackMixerElem "Front Headphone"
+        }
+}
+
+SectionDevice."SPDIF" {
+        Comment "S/PDIF Out"
+        Value {
+                PlaybackPriority 1
+                PlaybackChannels "2"
+                PlaybackPCM "hw:ALC1220VBDT,2"
+                PlaybackMixerElem "IEC958"
+        }
+}
+
+SectionDevice."Line" {
+        Comment "Line In"
+        Value {
+                CaptureChannels "2"
+                CapturePCM "hw:ALC1220VBDT,0"
+                JackControl "Line Jack"
+                CaptureMixerElem "Line"
+        }
+}
+
+SectionDevice."Mic" {
+        Comment "Microphone"
+        Value {
+                CaptureChannels "2"
+                CapturePCM "hw:ALC1220VBDT,1"
+                JackControl "Mic Jack"
+                CaptureMixerElem "Mic"
+        }
+}
+
+SectionDevice."FrontMic" {
```

Comment:
FrontMic is not a valid UCM device name. Use "Mic1" and "Mic2" and write the location to comments.